contract_compassion/model/contract_group.py

```diff
@@ -0,0 +1,246 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2014 Compassion CH (http://www.compassion.ch)
+#    Releasing children from poverty in Jesus' name
+#    @author: Cyril Sester, Emanuel Cino
+#
+#    The licence is in the file __openerp__.py
+#
+##############################################################################
+
+import time
+
+from openerp.osv import orm, fields
+from openerp.tools.translate import _
+from openerp.tools import mod10r
+from openerp import netsvc
+
+
+class contract_group(orm.Model):
+    ''' Add BVR on groups and add BVR ref and analytics_id

```

Comment:
Please use double quotes for docstrings as we always used double quotes... Replace everywhere you can find them.